subprojects/docs/src/docs/release/notes.md

```diff
@@ -50,6 +50,14 @@ Example:
 ADD RELEASE FEATURES BELOW
 vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
 
+### Wrapper task validates distribution url
+
+The wrapper task now validates the configured distribution url before writing it to the gradle-wrapper.properties file.
+If the url scheme is `http` or `https`, a HEAD request is sent.
+If the scheme is `file`, the existence of the file is checked.
+This surfaces invalid urls early and can prevent IO exceptions at execution time.
+
+More details can be found in the dedicated section of the [Gradle Wrapper](gradle_wrapper.html#[adding_the_gradle_wrapper](sec:adding_wrapper)) user manual chapter.
```

Comment:
:x: It would be better to have actually more details in the userguide than in release notes. Especially what the check is about depending on the URL scheme.